### ISSUE TITLE
[WCAG 2.2 Draft Feedback] Summarized Feedback

### ISSUE THREAD

**Issue by dshoukry** [w3c/wcag#2703]
As we raised in our previous reviews and 2020 charter response, we hope our constructive contributions towards the end of an arduous process provide some support. We appreciate that three of the success criteria we had significant concerns about were held back for more refinement since the last draft, and that so much of our other feedback was incorporated.

We still have some significant concerns about Dragging Movements, Target Size Minimum, and Focus Appearance Minimum, and recommend these to be held back and pursued in a future update. We believe more research is needed on multiple input modalities and ensure real world benefits to as many users as possible.  

We’ll provide feedback in a similar style to previous rounds (summarized feedback for each SC as well as a public Google Doc with comments, questions, and suggestions).

**Comment by aardrian** [comment]
I missed this meta issue or I would have left this feedback here instead of in one of the other issues...

Can you please copy the entirety of your Google Doc into each issue you filed today?

At the very least, understand that the accessibility of Google Docs is not ideal. Asking readers to switch back and forth is at least a frustration to participation in the discussion and at most a barrier. Never mind the genuine concern that the document will not be maintained in perpetuity (without being edited throughout this discussion).

As a reader who wants to see what issues have been raised, walling the comments off in a different platform means I also cannot perform text searches against the entirety of comments here.

In addition, some organizations (insert random financial institution here) block access to Google Docs from within their networks, making participation during work hours as part of employment impossible.

**Comment by alastc** [comment]
As there is no direct feedback in this issue I will close it and focus on the individual SC issues.
